.github/workflows/timeseries-scans.yml

```diff
@@ -65,37 +48,70 @@ jobs:
           curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/html.tpl -o trivy-html.tpl
           # Use the downloaded template
-          trivy fs . --format template --template "@trivy-html.tpl" -o "trivy_fs_code_scan.html"   
+          trivy fs . --format template --template "@trivy-html.tpl" -o "trivy_fs_code_scan.html" 
+          trivy fs --list-all-pkgs --format template --template "@trivy-html.tpl" --output trivy-fs-full-report.csv .
+          trivy fs --ignore-unfixed . | tee trivy-fs-full-report.txt  
       
-      - name: Upload trivy reports
+      - name: Upload trivy filesystem/repo scan reports
         continue-on-error: true
         uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: trivy-code-scan-results-core
+          name: Trivy FileSystem scan report
           path: |
               microservices/time-series-analytics/trivy_fs_code_scan.html
+              microservices/time-series-analytics/trivy-fs-full-report.csv
+              microservices/time-series-analytics/trivy-fs-full-report.txt
       
       - name: Trivy Image Scan
         continue-on-error: true
         shell: bash
         run: |
               pwd
-              curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/html.tpl -o trivy-html.tpl
               echo "Building Time Series Analytics scanning"
               cd microservices/time-series-analytics/docker
               sed -i -e "s|TIME_SERIES_ANALYTICS_IMAGE=.*|TIME_SERIES_ANALYTICS_IMAGE=ia-time-series-analytics-microservice:latest|g" .env
               docker compose build
+              curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/html.tpl -o trivy-html.tpl
               trivy image ia-time-series-analytics-microservice:latest --ignore-unfixed --format template --template "@trivy-html.tpl" -o trivy_image_scan.html
               trivy image --quiet --format spdx-json --output trivy_image_scan.spdx.json ia-time-series-analytics-microservice:latest
               echo "completed Time Series Analytics scanning"
             
-      - name: Upload Trivy Image Report
+      - name: Upload Trivy Image Scan Report
         continue-on-error: true
         if: always()
         uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02
         with:
-          name: Trivy image scan report-core
+          name: Trivy image scan report
           path: |
-                microservices/time-series-analytics/trivy_image_scan.html
-                microservices/time-series-analytics/trivy_image_scan.spdx.json
\ No newline at end of file
+                microservices/time-series-analytics/docker/trivy_image_scan.html
+                microservices/time-series-analytics/docker/trivy_image_scan.spdx.json
+
+      - name: Trivy file scan 
+        run: |
+          
+          
+      - name: Trivy config scan for Dockerfiles and helm charts 

```

Comment:
Since in this step we are running only for helm cahrts, let's update accordingly